mcs/build/tests.make

```diff
@@ -147,19 +97,36 @@ test-local-aot-compile: $(topdir)/build/deps/nunit-$(PROFILE).stamp
 
 endif # ALWAYS_AOT
 
+NUNITLITE_CONFIG_FILE=$(topdir)/class/lib/$(PROFILE)/nunit-lite-console.exe.config
```

Comment:
Change it to `$(topdir)/class/lib/$(PROFILE)/$(PARENT_PROFILE)nunit-lite-console.exe.config` and you could remove the ifdef bellow